docs/api/saved-objects/export.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[saved-objects-api-export]]
+=== Export Objects
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The export saved object API enables you to retrieve a set of Kibana saved objects by various conditions into a newline delimited JSON format.
+
+Note: You cannot access this endpoint via the Console in Kibana.
+
+==== Request
+
+`GET /api/saved_objects/_export`
+
+==== Query Parameters
+`type` (optional)::
+  (array|string) The saved object type(s) that the export should be limited to
+`objects` (optional)::
+  (array) A list of objects to export
```

Comment:
How to indicate at least `type` or `objects` needs to be passed in and not both at the same time?